src/pages/cardsPage/CardsPage.jsx

```diff
@@ -0,0 +1,59 @@
+import { React, useState, useEffect } from 'react';
+import { Box, TextField, ThemeProvider } from '@mui/material';
+import { UserCard } from '../../components/UserCard/UserCard';
+import theme from './theme';
+import './CardsPageStyles.css';
+import fetchUserData from '../../api/usersApi';
+import useLocalStorage from '../../customHooks/useLocalStorage';
+
+function CardsPage() {
+
+  const [users, setUsers] = useState([]);
+  const [search, setSearch] = useState('');
+  const [favorites, setFavorites] = useLocalStorage([]);
+
+  useEffect(() => {
+    fetchUserData().then(fetchedData => setUsers(fetchedData));
+  }, []);
+
+  const handleInputChange = (event) => {
+    setSearch(event.target.value);
+  }
+
+  const filterUsers = (users, search) => {
+    return users.filter((user) => {
+      if (users.length !== 0 && search === '') {
+        return user;
+      } else if (user.name.toLowerCase().includes(search.toLowerCase())) {
+        return user;
+      } else {
+        return null;
+      }
+    });
+  }
+
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
The Theme should have all of the pages, so this is not for here. Add the theme one level above in the App.